src/components/custom-button-link/custom-button-link.tsx

```diff
@@ -1,29 +1,11 @@
 import React from 'react';
-import {
-  TouchableOpacity,
-  StyleSheet,
-  TouchableOpacityProps,
-} from 'react-native';
+import {TouchableOpacity, TouchableOpacityProps} from 'react-native';
 import * as Styled from './custom-button-link.styles';
```

Comment:
esse arquivo exporta somente um componente, fica mais facil vc importar ele direto aqui n?
```suggestion
import { LabelCustomButtonLink } from './custom-button-link.styles';
```

Reply:
beleza, eu tinha usado o styled. antes para ficar padronizado o código, irei mudar aqui